server/services/utils/formatForView.js

```diff
@@ -1,12 +1,16 @@
 const moment = require('moment');
 const setCase = require('case');
-module.exports = {formatObjectForView};
+module.exports = {formatObjectForView, formatObjectForViewWithOptions};
 
-function formatObjectForView(object, options) {
+function formatObjectForView(object) {
+    return formatObjectForViewWithOptions(object, {capitalise: nameFields, dates: dateFields, custom: customFields});
+}
```

Comment:
Not sure there is any value in a function that just calls another one. Would have been tempted to just have one function as all we need passing in is the object itself (as options don't seem to change at this point)